onnxruntime/core/providers/cuda/tensor/split_impl.cu

```diff
@@ -8,46 +8,49 @@
 namespace onnxruntime {
 namespace cuda {
 
-template <typename T>
+template <typename T, typename T_INT64, typename T_OUTPUT>
```

Comment:
>T_INT64 [](start = 31, length = 7)

how about something like `TInt64Array` and `TVoidPtrArray` to convey that we use operator[]? the current names are not that intuitive to me